patches/chromium/replace_chrome_grit_with_electron_grit_in_native_desktop_media_list.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jeremy Rose <jeremya@chromium.org>
+Date: Tue, 24 Aug 2021 15:07:03 -0700
+Subject: replace chrome grit with electron grit in native_desktop_media_list
+
+This is to allow looking up of a couple of IDS_* strings which this part
+of Chrome depends on.
+
+diff --git a/chrome/browser/media/webrtc/native_desktop_media_list.cc b/chrome/browser/media/webrtc/native_desktop_media_list.cc
+index 5a851f4c89b4ed2b41640bf8bad47b7d8eba8ca1..ddc91cc22dd073cf9ee005f4e22a20443d41f439 100644
+--- a/chrome/browser/media/webrtc/native_desktop_media_list.cc
++++ b/chrome/browser/media/webrtc/native_desktop_media_list.cc
+@@ -16,7 +16,7 @@
+ #include "base/threading/thread_task_runner_handle.h"
+ #include "build/build_config.h"
+ #include "chrome/browser/media/webrtc/desktop_media_list.h"
+-#include "chrome/grit/generated_resources.h"
++#include "electron/grit/electron_resources.h"
+ #include "content/public/browser/browser_task_traits.h"
+ #include "content/public/browser/browser_thread.h"
+ #include "content/public/common/content_features.h"
```

Comment:
Would it be better to add this to https://github.com/electron/electron/blob/main/patches/chromium/fix_use_electron_generated_resources.patch instead of adding another patch?